compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/types/BLangTupleTypeNode.java

```diff
@@ -37,11 +37,11 @@
 public class BLangTupleTypeNode extends BLangType implements TupleTypeNode {
 
     // BLangNodes
-    public List<BLangType> memberTypeNodes = new ArrayList<>();
+    public List<BLangMemberTypeNode> memberTypeNodes = new ArrayList<>();
```

Comment:
I see we have initialized here as well. But we can keep this as it will be a somewhat significant change unrelated to this PR.